roles/openshift_facts/library/openshift_facts.py

```diff
@@ -317,9 +317,8 @@ def set_fluentd_facts_if_unset(facts):
 
     """
     if 'common' in facts:
-        deployment_type = facts['common']['deployment_type']
         if 'use_fluentd' not in facts['common']:
-            use_fluentd = True if deployment_type == 'online' else False
+            use_fluentd = False
```

Comment:
In the future changes to the online defaults will go through the release process(ie issue on the release ticket or PR against the release branch), until then IMHO they should be set by the OPs team.